mathics/core/definitions.py

```diff
@@ -100,8 +100,8 @@ def __init__(
         self._packages = []
         self.current_context = "Global`"
         self.context_path = (
-            "Global`",
             "System`",
+            "Global`",
```

Comment:
Do I understand correctly, that are context path had been wrong previously? 

Other than this message problem,  how can we tell? 

Thanks in advance for the education here.

Reply:
Yep: in WMA, "System`" is before that "Global`" in ContextPath, and in the order of name resolution.

Reply:
Ok - thanks for the clarification. Given this, it probably should me mentioned in the CHANGES.rst as this is kind of a big deal.  (And the name of the PR doesn't give any clue that this is going on either.)